master/internal/api_project_intg_test.go

```diff
@@ -32,8 +32,10 @@ import (
 
 var pAuthZ *mocks.ProjectAuthZ
 
+const mockName = "mock"
```

Comment:
note: had to add these changes to satisfy the linter, although my PR doesn't touch this